### ISSUE TITLE
query string parameters values are truncated

### ISSUE THREAD

**Issue by leroyleejh** [awslabs/aws-sam-cli#736]
Sending parameters in my query strings results in only the last character being sent in `event.queryStringParameters`

Example browser access
```
localhost:3000/hello?message1=hello&message2=world
```

Event sent to lambda:
```
{
  "httpMethod": "GET",
  "body": null,
  "resource": "/hello",
  "requestContext": {
    "resourceId": "123456",
    "apiId": "1234567890",
    "resourcePath": "/hello",
    "httpMethod": "GET",
    "requestId": "c6af9ac6-7b61-11e6-9a41-93e8deadbeef",
    "accountId": "123456789012",
    "stage": "prod",
    "identity": {
      "apiKey": null,
      "userArn": null,
      "cognitoAuthenticationType": null,
      "caller": null,
      "userAgent": "Custom User Agent String",
      "user": null,
      "cognitoIdentityPoolId": null,
      "cognitoAuthenticationProvider": null,
      "sourceIp": "127.0.0.1",
      "accountId": null
    },
    "extendedRequestId": null,
    "path": "/hello"
  },
  "queryStringParameters": {
    "message1": "o",
    "message2": "d"
  },
  "headers": {
    "Host": "localhost:3000",
    "Connection": "keep-alive",
    "Upgrade-Insecure-Requests": "1",
    "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36",
    "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8",
    "Accept-Encoding": "gzip, deflate, br",
    "Accept-Language": "en-GB,en-US;q=0.9,en;q=0.8",
    "X-Forwarded-Proto": "http",
    "X-Forwarded-Port": "3000"
  },
  "pathParameters": null,
  "stageVariables": null,
  "path": "/hello",
  "isBase64Encoded": false
}
```

**Observed result:**
```
"queryStringParameters":{  
      "message1":"o",
      "message2":"d"
   }
```
Is only returning the last character of the each param in the query string

**Expected result:**
```
"queryStringParameters":{  
      "message1":"hello",
      "message2":"world"
   }
```

**Additional environment details (Ex: Windows, Mac, Amazon Linux etc)**
Using 
Docker Image `lambci/lambda:nodejs8.10`
Mac `High Sierra 10.13.6`

**Output of `sam --version`**:
```
SAM CLI, version 0.6.1
```

**Comment by jfuss** [comment]
@leroyleejh What python version did you install SAM CLI with? If you are using Python3.6.7 or 3.7.1, we found a change in behavior with how `dict()` works. We have merged and patched/fixed this in #679.

**Comment by leroyleejh** [comment]
@jfuss I installed using Python 3.7.1. Will reinstall SAM CLI with 2.7 and test again. Thank you for the response!

**Comment by leroyleejh** [comment]
Update: I have downgraded Python to 3.7.0, and reinstalled SAM CLI.
It is working fine now.

**Comment by jfuss** [comment]
This was released with [v0.6.2](https://github.com/awslabs/aws-sam-cli/releases/tag/v0.6.2). 

Closing
